scripts/get_all_heron_paths.sh

```diff
@@ -23,7 +23,19 @@ function query() {
 
 set +e
 # Build everything
-bazel build {heron,integration-test,tools/java}/...
+os_type=darwin
+case "$OSTYPE" in
```

Comment:
instead of requiring the user to pass this, could we instead try to auto-detect it using `uname`?